cg/meta/archive/archive.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from collections import namedtuple
+from typing import Dict, List
+
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.meta.archive.ddn_dataflow import DDNDataFlowApi
+from cg.store import Store
+
+LOG = logging.getLogger(__name__)
+
+PathAndSample = namedtuple("PathAndSample", "path sample_internal_id")
+
+
+class ArchiveAPI:
+    """Class handling the archiving of sample SPRING files."""
+
+    def __init__(
+        self, ddn_dataflow_api: DDNDataFlowApi, housekeeper_api: HousekeeperAPI, status_db: Store
+    ):
+        self.housekeeper_api: HousekeeperAPI = housekeeper_api
+        self.ddn_api: DDNDataFlowApi = ddn_dataflow_api
+        self.status_db: Store = status_db
+
+    def sort_files_on_archive_location(
```

Comment:
Maybe `sort_files_by_archive_location`, feel free to disregard

Reply:
I agree we sort by